plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -118,6 +118,9 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject,
       subMonitor.worked(1);
     }
     javaProject.setRawClasspath(newRawClasspath, monitor);
+    
+    Job job = new AppEngineLibraryContainerResolverJob("Initialize libraries", javaProject);
```

Comment:
Performing this as a job seems a teeny bit odd considering this is already being done in a forked-off operation.  But it does get the project-creation wizard to disappear faster.